src/view/com/util/EmptyState.tsx

```diff
@@ -1,76 +1,115 @@
-import {StyleProp, StyleSheet, View, ViewStyle} from 'react-native'
-import {IconProp} from '@fortawesome/fontawesome-svg-core'
+import React from 'react'
+import {type StyleProp, type TextStyle, type ViewStyle} from 'react-native'
+import {StyleSheet, View} from 'react-native'
+import {type IconProp} from '@fortawesome/fontawesome-svg-core'
 import {
   FontAwesomeIcon,
-  FontAwesomeIconStyle,
+  type FontAwesomeIconStyle,
 } from '@fortawesome/react-native-fontawesome'
 
 import {usePalette} from '#/lib/hooks/usePalette'
 import {useWebMediaQueries} from '#/lib/hooks/useWebMediaQueries'
 import {UserGroupIcon} from '#/lib/icons'
+import {atoms as a} from '#/alf'
+import {Button, type ButtonProps, ButtonText} from '#/components/Button'
 import {Growth_Stroke2_Corner0_Rounded as Growth} from '#/components/icons/Growth'
-import {Text} from './text/Text'
+import {Text} from '#/components/Typography'
+
+export type EmptyStateButtonProps = Omit<ButtonProps, 'children' | 'label'> & {
+  label: string
+  text: string
+}
 
 export function EmptyState({
   testID,
   icon,
   message,
   style,
+  textStyle,
+  button,
 }: {
   testID?: string
-  icon: IconProp | 'user-group' | 'growth'
+  icon: IconProp | 'user-group' | 'growth' | React.ReactElement
   message: string
   style?: StyleProp<ViewStyle>
+  textStyle?: StyleProp<TextStyle>
+  button?: EmptyStateButtonProps
 }) {
   const pal = usePalette('default')
   const {isTabletOrDesktop} = useWebMediaQueries()
   const iconSize = isTabletOrDesktop ? 64 : 48
+
+  const renderIcon = () => {
+    if (typeof icon === 'object' && React.isValidElement(icon)) {
+      return icon
+    }
+
+    if (icon === 'user-group') {
```

Comment:
what are these used for?

Reply:
It looks like the `user-group` is no longer being used even though it was used in the past so I can remove it, and for the "growth" we can remove it as well once the Notifications empty state is being worked on either in this or a follow up PR.